Tools/ChangeLog

```diff
@@ -1,3 +1,23 @@
+2022-05-11  Kate Cheney  <katherine_cheney@apple.com>
+
+        Need a short description (OOPS!).
+        Need the bug URL (OOPS!).
+        rdar://45454933
+
+        Reviewed by NOBODY (OOPS!).
+
+        API test coverage.
+
+        * TestWebKitAPI/Tests/mac/ContextMenuTests.mm:
+        (TestWebKitAPI::rightClick):
+        For whatever reason the webview in the API test seems to need one click
+        before the container node for the image is not null, so this test
+        executes two right clicks then checks for the presence of a context
+        menu and whether the image is selected. This prevents a false positive
+        result when the change is not present.
```

Comment:
Perhaps this is just because the selection is none in this case? (e.g. not collapsed, and also not a range).

If you prefer, you might be able to evaluate `getSelection().setPosition(document.body);` and then just right click once, instead of right clicking twice. That said, right clicking twice is fine too!

Reply:
Ooh that makes sense. Thanks for the review!